### PR TITLE
docs: add notebook

### PR DESCRIPTION
**Pull request by PierreLeveau**

### CODE REVIEW

File: recipes/finetuning_openai_summaries.ipynb [hunk 1]
```diff
@@ -0,0 +1,1723 @@
+{
```
Comment (Jonas1312) [inline recipes/finetuning_openai_summaries.ipynb:1]: <p>4) from the datset</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Jonas1312) [inline recipes/finetuning_openai_summaries.ipynb:1]: <p>we should explain that the other summaries are generated using a temperature of 1 to have some randomness, whereas the first generation has a temperature of 0</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (Jonas1312) [inline recipes/finetuning_openai_summaries.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.    disk_cache = dc.Cache("cache")
</pre><p> </p><p>do we really need disk cache for this? Can't we use lru_cache directly?</p><p> </p><p>We should explain why we use a cache</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (Jonas1312) [inline recipes/finetuning_openai_summaries.ipynb:1]: <p>clear cell output</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (Jonas1312) [inline recipes/finetuning_openai_summaries.ipynb:1]: <p>the plot is hard to interpret</p><p> </p><p>we should change it or add some sentences to help </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Review by Jonas1312 — CHANGES_REQUESTED** [review]
comments above

File: recipes/finetuning_openai_summaries.ipynb (hunk 1 above)
Comment (PierreLeveau) [reply recipes/finetuning_openai_summaries.ipynb:1]: it was only to accelerate the development, I am removing it

**Review by Jonas1312 — APPROVED** [review]

File: recipes/finetuning_openai_summaries.ipynb (hunk 1 above)
Comment (PierreLeveau) [reply recipes/finetuning_openai_summaries.ipynb:1]: changed

Comment (PierreLeveau) [reply recipes/finetuning_openai_summaries.ipynb:1]: cleared

**Pull request merged**